edgexfoundry/device-sdk-go
# refactor: Use go-mod-bootstrap RequestLimitMiddleware for MaxRequestSize

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-sdk-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?)n/a, replace function
- [ ] I have opened a PR for the related docs change (if not, why?) n/a, replace function
  <link to docs PR>

## Testing Instructions

1. build the example device-simple from this branch
2. set MaxRequestSize to 1 (kb)
3. send PUT request with a large request body and verify the response
## New Dependency Instructions (If applicable)
